site/includes/footerresources.hbs

```diff
@@ -1,5 +1,6 @@
 <!--[if gte IE 9 | !IE ]><!-->
 <script src="{{#if offline}}{{assets}}/../wet-boew/js{{else}}https://ajax.googleapis.com/ajax/libs{{/if}}/jquery/{{environment.jqueryVersion}}/jquery.js"></script>
+<script>window.jQuery || document.write('<script src="{{assets}}/../wet-boew/js/jquery/{{environment.jqueryVersion}}/jquery.js"><\/script>');</script>
```

Comment:
Had a quick offline discussion with @laurentgoderre. I will remove the line when I have access to my laptop. Currently out of the office unexpectedly but I'll be able to get something sorted before end of business day.